### DISCUSSION TITLE
Feature Request: Time Series Graph - Grid Style Settings

### DISCUSSION THREAD

**Discussion by jasonacox** [grafana/grafana#56431] [Dashboards, Panels & Visualizations]
## Problem to Solve

The grid lines on the new "time series" graphs are difficult to see.  The old time series graphs had a more defined and visible grid.  It is likely a personal preference or based on the type of data being visualized.  It would be great to be able to adjust the grid attributes (e.g. gridColor). 

## Examples

Old TS:
<img width="536" alt="image" src="https://user-images.githubusercontent.com/836718/194211180-73ff1e26-b247-4d71-8ec0-52260533a19a.png">

New TS (grid is faded and harder to see)
<img width="542" alt="image" src="https://user-images.githubusercontent.com/836718/194211238-a8ac5013-16bd-4a7a-bf48-b2c1cc1299f4.png">

## Research

I didn't find any user control to adjust the color of the gridlines but did see that there appears to be a hardcoded #c8c8c8 as part of this commit: https://github.com/grafana/grafana/commit/1daa0dfe653002dff991f607db38f22ba314f3af

Could we make the gridColor a user editable value?  Or is the color the same here and some anti-aliasing work causing the fading?  In any case, it would be great to have an option to change the color, 'unfade' and possibly set other grid line styling attributes. 😁  

Thanks for a great project!  ❤️  Grafana.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
